core/backend/src/GeographicCRSServices.ts

```diff
@@ -38,6 +38,10 @@ export interface GetAvailableCoordinateReferenceSystemsArgs {
    * Maximum longitude and latitude correspond to extent.high.x and extent.high.y, respectively.
    */
   extent?: Range2dProps;
+  /** If true, returns additional CRS with extents spanning the entire Earth's surface.
```

Comment:
I don't like using "CRS" in a way where it is supposed to be plural. The `extent` comment above uses "CRS's", which I also don't particularly like, but I feel that it is better than just "CRS". (Grammatically, I don't think "apostrophe s" is ever allowed to indicate plural, but with acronyms, it might be accepted.) Whatever is used should be consistent. I feel that the only real options are "CRS's" and "Coordinate Reference Systems" in both locations.

Reply:
> don't think "apostrophe s" is ever allowed to indicate plural, but with acronyms, it might be accepted.

Seems to be bending toward allowing in some cases, but I agree. Looking it up now, it seems CRSs might be the best idea, which I don't hate.

Reply:
I'm ok with CRSs as long as you update the `extent` comment to be consistent.

Reply:
I've moved to coordinate reference systems in that file for consistency... we're all over the place across the repo.